Dockerfile

```diff
@@ -276,6 +276,10 @@ RUN pip install --upgrade cython && \
     pip install google-cloud-language==2.* && \
     pip install google-cloud-videointelligence==2.* && \
     pip install google-cloud-vision==2.* && \
+    # After launch install main
+    pip install git+https://github.com/googleapis/python-aiplatform.git@mb-release && \ 
```

Comment:
Is there a prebuilt version of the python aiplatform on pip that we could use?

Reply:
there isn't unfortunately, the dev branches has the needed changes and won't be merged until launch